lib/building.rb

```diff
@@ -23,14 +19,12 @@ def total_annual_rent
 
   def occupancy
     if units.empty?
-      return @occupancy
+      0.0
     else
-      (number_of_occupied_units.to_f / units.count.to_f ) * 100
+      (number_of_occupied_units / units.count.to_f ) * 100
```

Comment:
only the second thing needs to be `to_f`, but you might run into this concept in black thursday and decimals. This doesn't prevent it from returning 33.33333333333333333333333. And you might want to truncate the number somehow?